server/config/config.go

```diff
@@ -201,6 +203,33 @@ func NewConfig() *Config {
 	return cfg
 }
 
+// ServiceMode is the mode of the service.
+type ServiceMode int
+
+const (
+	APIService ServiceMode = iota
+	TSOService
+	ResourceManagerService
+	SchedulerService
+	ServiceModeCount
+)
+
+// String
+func (s ServiceMode) String() string {
```

Comment:
need to update the "services mode" section in "RFC: PD-Server supports multi-services mode" to make sure they are in sync.